src/app/docs/vertical-tabs/vertical-tabs-docs.component.html

```diff
@@ -0,0 +1,272 @@
+<sky-docs-demo-page moduleName="SkyTabsModule" moduleSourceCodePath="src/app/public/modules/vertical-tabset/"
+  pageTitle="Vertical tabs">
+  <sky-docs-demo-page-summary>
+    Vertical tabs divide and categorize page content when the number of categories is large.
+  </sky-docs-demo-page-summary>
+
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel (selectionChange)="onDemoSelectionChangeTabs($event)">
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox label="Include counts" propertyName="counts" [checked]="false">
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+
+    <sky-vertical-tabset>
+      <sky-vertical-tab *ngFor="let tab of tabs" [active]="tab.active" [tabHeading]="tab.tabHeading"
+        [tabHeaderCount]="tab.tabHeaderCount" [disabled]="tab.disabled">
+        {{ tab.content }}
+      </sky-vertical-tab>
+    </sky-vertical-tabset>
+  </sky-docs-demo>
+
+  <sky-docs-demo>
+    <sky-docs-demo-control-panel (selectionChange)="onDemoSelectionChangeGroups($event)">
+      <sky-docs-demo-control-panel-section>
+        <sky-docs-demo-control-panel-checkbox label="Include counts" propertyName="counts" [checked]="false">
+        </sky-docs-demo-control-panel-checkbox>
+      </sky-docs-demo-control-panel-section>
+    </sky-docs-demo-control-panel>
+
+    <sky-vertical-tabset showTabsText="Tab list">
+
+      <sky-vertical-tabset-group *ngFor="let group of groups" [groupHeading]="group.heading" [open]="group.isOpen"
+        [disabled]="group.isDisabled">
+
+        <sky-vertical-tab *ngFor="let tab of group.subTabs" [active]="tab.active" [tabHeading]="tab.tabHeading"
+          [tabHeaderCount]="tab.tabHeaderCount" [disabled]="tab.disabled">
+          {{ tab.content }}
+        </sky-vertical-tab>
+      </sky-vertical-tabset-group>
+    </sky-vertical-tabset>
+  </sky-docs-demo>
+
+  <sky-docs-design-guidelines>
+    <sky-docs-demo-page-section heading="Usage">
+      <sky-docs-design-guideline heading="Use when" headingStyle="success">
+        <p>
+          Use vertical tabs when the number of tabs and/or the tab label text is expected to exceed the horizontal space
+          available.
+        </p>
+        <sky-docs-design-guideline-thumbnail
```

Comment:
this example looks like a split view to me

Reply:
hmm, I can see that. Is it the header giving that impression? Should I try something less temporary and with less workflow implication?

Reply:
more the labels on the tabs. it reads like a list of items you are working through.

Reply:
Do you think changing the label on the list itself can help? If we call these "Affiliates" or "Affiliated organizations", does the list appear more static and less like the kind of to-do list that would warrant a Split view?

Reply:
@Blackbaud-AdamFunderburk and @Blackbaud-ToddRoberts, can we resolve this conversation? Or are changes still pending?

Reply:
I've updated the visual example vertical-tabs-usage-1